### PR TITLE
Fix up build the docs site

### PR DESCRIPTION
**Pull request by rhatdan**
[NO TESTS NEEDED]

Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

### CODE REVIEW

**Comment by rhatdan** [comment]
Alternative to https://github.com/containers/podman/pull/11611

**Comment by rhatdan** [comment]
@TomSweeneyRedHat This will fix up the man pages to show the full podman-machine.1 rather then just the subcommands.

**Comment by rhatdan** [comment]
@edsantiago @baude @Luap99 @flouthoc PTAL

**Comment by edsantiago** [comment]
This seems like it will just make *all* the man pages broken?

**Comment by rhatdan** [comment]
Why? I think the tests are broken.  Looking at this from a website looks perfect.

**Comment by Luap99** [comment]
I am fine with this change but this will not fix the "links in markdown tables" problem. To fix the links https://github.com/containers/podman/pull/11614 is needed.

File: docs/source/Commands.rst [hunk 1]
```diff
@@ -13,7 +13,7 @@ Commands
 
 :doc:`commit <markdown/podman-commit.1>` Create new image based on the changed container
 
-:doc:`container <managecontainers>` Manage Containers
+:doc:`container <markdown/podman-containers.1>` Manage Containers
```
Comment (Luap99) [inline docs/source/Commands.rst:16]:
```suggestion
:doc:`container <markdown/podman-container.1>` Manage Containers
```

**Review by Luap99 — CHANGES_REQUESTED** [review]

**Review by Luap99 — APPROVED** [review]
LGTM

**Review by giuseppe — APPROVED** [review]
/lgtm

**Comment by rhatdan** [comment]
@flouthoc @saschagrunert @giuseppe @containers/podman-maintainers PTAL

**Pull request merged**